### ISSUE TITLE
Fluxion 4.12 stucks on ..

### ISSUE THREAD

**Issue by Dylan433447** [FluxionNetwork/fluxion#548]
Hello ! 

I've been using Fluxion latest version and i got a bug here 
when i use Evil Twins --- HOSTPAD ---- IT STUCKS LIKE THIS
[fluxion@DyLaN]-[~] 1

[*] Starting Captive Portal access point service...

My info : 
using LAPTOP INTERNAL WIRELESS CARD 
WIFI :  Qualcomm Atheros AR9285 Wireless Network Adapter (PCI-Express)

**Comment by usama7628674** [comment]
@Dylan433447 See #509

**Comment by Dylan433447** [comment]
BUG: unable to handle kernel NULL pointer dereference at 0000000000000066

i also get this error 


You mean i should downgrade my kernel ?  is that right ? 

could anybody tell me how to downgrade my kernel ???? if that right ?

or Could you guys tell me any solution for this issue ??

**Comment by usama7628674** [comment]
Just download kali 2018.1 and run the script again and don't upgrade your kernal again for the time being.

**Comment by MPX4132** [comment]
Sounds like a problem with your system, maybe a driver, but fluxion doesn’t throw errors like that.

**Comment by Dylan433447** [comment]
Usama ? What u mean by downloading 2018.1 again !! U mean reinstall kali !!

**Comment by usama7628674** [comment]
@Dylan433447 Yes download and install kali 2018.1 and you won't receive this error.

**Comment by Dylan433447** [comment]
HOW TO PREVENT UPGRADING KERNEL?

**Comment by usama7628674** [comment]
@Dylan433447 Do what @rad4day said.You just have to prevent all packages which have 4.16.12 in it.

**Comment by Dylan433447** [comment]
So i have to google how to block linux-* and linux headers packages from updating ?

**Comment by 3mmory** [comment]
i have this problem in launching the program            
[*] aircrack-ng...................................... OK.           
            [*] python2.......................................... OK.           
            [*] bc.......................................... Missing!           
            [*] awk.............................................. OK.           
            [*] curl............................................. OK.           
            [*] cowpatty......................................... OK.           
            [*] dhcpd....................................... Missing!           
            [*] 7zr.............................................. OK.           
            [*] hostapd..................................... Missing!           
            [*] lighttpd.................................... Missing!           
            [*] iwconfig......................................... OK.           
            [*] macchanger....................................... OK.           
            [*] mdk3............................................. OK.           
            [*] nmap............................................. OK.           
            [*] openssl.......................................... OK.           
            [*] php-cgi..................................... Missing!           
            [*] pyrit............................................ OK.           
            [*] xterm............................................ OK.           
            [*] rfkill........................................... OK.           
            [*] unzip............................................ OK.           
            [*] route............................................ OK.           
            [*] fuser............................................ OK.           
            [*] killall.......................................... OK.           
Nothing to check.
Adding missing sources to package manager, please wait.
Adding keys.gnupg.net key, please wait.
**then it says :** 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package bc

Dependency installation failed!
Press enter to retry, ctrl+c to exit...

**Comment by iToXiC-Devex** [comment]
I have the same issue... fluxion was working fine, then suddenly it started stucking at “Starting Captive Portal Service” any help? I have the same wireless adapter as Dylan433447 which is AR9285. I was using fluxion with the same adapter tho and it was working fine...

**Comment by usama7628674** [comment]
@Didn't you read the hole thread?See #509

**Comment by iToXiC-Devex** [comment]
@usama7628674 so basically i need to reinstall kali?

**Comment by usama7628674** [comment]
Exactly

**Comment by iToXiC-Devex** [comment]
@usama7628674 sorry for asking many questions... but i just want to know should it be a different version? Or i can just reinstall the same version again?

**Comment by usama7628674** [comment]
@iToXiC-Devex Just download kali 2018.1 and don't update it or update packages only and don't do `apt dist-upgrade`

**Comment by 3x-M4ch1nA** [comment]
How to fix fluxion issues? Downgrade your distro because fluxion can't keep up to date enough to solve the issue on their end. Seems legit

**Comment by usama7628674** [comment]
@3x-M4ch1nA You're totally wrong.The bug is in kernal not fluxion.

**Comment by MPX4132** [comment]
@3x-M4ch1nA Maybe if you stopped for a bit and helped out, we might be able to "fix fluxion", man.

**Comment by rad4day** [comment]
@Dylan433447 ask google or on stackexchange. You should find a how to for your package manager / distro.
Also it's pretty annoying for all people who are on the mailing list for this issue to receive comments after almost one month. That's why I'm locking it.
